### PR TITLE
First version of PostgreSQL role imported from plebia-configuration.

### PR DESCRIPTION
**Pull request by smarnach**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+POSTGRES_OPENSTACK_DB_DEVICE: "/dev/vdb1"
+POSTGRES_SERVER_BACKUP_DIR: '/var/lib/postgresql/backup'
```
Comment (jbzdak) [inline defaults/main.yml:2]: Minor, minor nit: Mixed `"` and `'` in a single file.

File: templates/backup-pre-postgres.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+TMPDIR="{{ POSTGRES_SERVER_BACKUP_DIR }}"
+
+rm -rf "$TMPDIR"
+mkdir -p "$TMPDIR"
+sudo -u postgres pg_dumpall > "$TMPDIR/postgresql_all_dbs.sql"
```
Comment (jbzdak) [inline templates/backup-pre-postgres.sh]: We might consider splitting the backups to match `mysql` role (restoring single database might be easier then). It's not really a requirement, rather just point that two database roles do things differently here.

Comment (antoviaque) [inline templates/backup-pre-postgres.sh]: +1 to splitting - managing/restoring one giant dump for all DBs can be pretty painful.

**Comment by jbzdak** [comment]
:+1:

**Pull request merged**